src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3259,20 +3259,17 @@ internal static void SetSessionStateDrive(ExecutionContext context, bool setLoca
 
```

Comment:
@iSazonov, your last commit had 91 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Clean up open Runspaces when exit powershell process.PowerShell process should not freeze at exit
```powershell
This command cannot be run due to the error: The directory name is invalid..
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Api\BasicEngine.Tests.ps1: line 52
52:         $process = Start-Process pwsh -ArgumentList $command -PassThru -RedirectStandardOutput $outputFile

```
Cdxml cmdlets are supported.Module level tests.The CimTest module should have the proper cmdlets
```powershell
MofComp.exe failed with exit code 
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Cdxml\Cdxml.Tests.ps1: line 98
98:             throw "MofComp.exe failed with exit code $MofCompReturnCode"

```
Cdxml cmdlets are supported.Get-CimTest cmdlet.The Get-CimTest cmdlet should return 4 objects
```powershell
MofComp.exe failed with exit code 
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Cdxml\Cdxml.Tests.ps1: line 98
98:             throw "MofComp.exe failed with exit code $MofCompReturnCode"

```
Cdxml cmdlets are supported.Get-CimTest cmdlet.The Get-CimTest cmdlet should retrieve an object via id
```powershell
MofComp.exe failed with exit code 
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Cdxml\Cdxml.Tests.ps1: line 98
98:             throw "MofComp.exe failed with exit code $MofCompReturnCode"

```
Cdxml cmdlets are supported.Get-CimTest cmdlet.The Get-CimTest cmdlet should retrieve an object by piped id
```powershell
MofComp.exe failed with exit code 
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Cdxml\Cdxml.Tests.ps1: line 98
98:             throw "MofComp.exe failed with exit code $MofCompReturnCode"

```